### ISSUE TITLE
Grindstone Disenchanting Incompatibility With Fabrication

### ISSUE THREAD

**Issue by scottwilson0308** [svenhjol/Charm#564]
Charm 2.3.2, Minecraft 1.16.5, Fabric Loader 0.11.3

Fabrication and Charm both allow the removal of enchantments via the Grindstone using books. Here's what happens with the following Grindstone disenchanting configs enabled:

Charm enabled, Fabrication enabled: enchanted tool and book can be placed, only disenchanted tool is output. Book is lost.
Charm enabled, Fabrication disabled: book with all enchantments is output. Tool is lost. (seems fair)
Charm disabled, Fabrication enabled: book cannot be placed in the lower slot at all. Vanilla Grindstone behaviour.
Charm removed, Fabrication enabled: Both the disenchanted tool and book with all enchantments are output.

TL;DR:

Charm must be removed entirely to use Fabrication's Grindstone Disenchanting feature. Disabling the Charm feature isn't enough.
Fabrication's same feature must be disabled for Charm's to work at all.

**Comment by svenhjol** [comment]
Priority 1.17 compat when there's an updated version.
Also tagging 1.16 support
